src/main/scala/org/ergoplatform/modifiers/mempool/ErgoTransaction.scala

```diff
@@ -285,7 +284,7 @@ object ErgoTransaction extends ApiCodecs with ScorexLogging with ScorexEncoding
   def extractAssets(boxes: IndexedSeq[ErgoBoxCandidate]): Try[(Map[ByteArrayWrapper, Long], Int)] = Try {
     val map: mutable.Map[ByteArrayWrapper, Long] = mutable.Map[ByteArrayWrapper, Long]()
     val assetsNum = boxes.foldLeft(0) { case (acc, box) =>
-      require(box.additionalTokens.length <= ErgoTransaction.MaxAssetsPerBox, "too many assets in one box")
+      require(box.additionalTokens.length <= MaxAssetsPerBox, "too many assets in one box")
```

Comment:
This change is causing hard-fork, as old nodes and ones with this code will diverge. 

Please leave verification protocol (the Ergo protocol) intact

Reply:
Fixed, I just removed the object reference `ErgoTransaction.` as we are withing the object scope ...